### DISCUSSION TITLE
Wrong api key

### DISCUSSION THREAD

**Discussion by knomdivad** [simple-login/app#1942] [Q&A]
I have seen a couple other items regarding this, but so far I have not seen a resolution. I am trying to write a python script to list out my aliases. Other functions will come later. 

        page = 0
        while True:
            url = f"https://app.simplelogin.io/api/aliases?page_id={page}"
            print("API Key: ", api_key)
            headers = {"Authorization": f"Bearer {api_key}"}
            response = requests.get(url, headers=headers)
        
The print function shows it is the correct API key when called. But then I immediately get:

Error fetching aliases: 401 {"error":"Wrong api key"}

I have tried several new API keys. And I have tried changing the Authroization header to just:

         headers = {"Authorization": f"{api_key}"}
         
Nothing seems to work.

**Comment by nguyenkims** [comment]
Your header isn't correct, it should use "Authentication" header instead. The below code should work:

```
api_key="your api key here"
requests.get("https://app.simplelogin.io//api/v2/aliases?page_id=1", headers={"Authentication": api_key})
```
